__tests__/storage.test.js

```diff
@@ -213,13 +213,46 @@ describe('Tozny', () => {
     const deleted = await ops.deleteAnonymousNote(written.noteId, signingPair)
     expect(deleted).toBe(true)
   })
-  it('can create groups', async () => {
+  it('can create, read and list groups', async () => {
```

Comment:
Make each of these a separate test, generally we want each test to be scoped to testing a single higher level concept, that way if the test fails its clear exactly what functionality of the api is broken / has regressed.